portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,52 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Sharon Dong</title>
     <link rel="stylesheet" href="style.css">
+    <link href='https://fonts.googleapis.com/css?family=Work Sans' rel='stylesheet'>    
     <script src="script.js"></script>
   </head>
+  <div class="topnav">
```

Comment:
Should this div belong to either body or head?

Reply:
I will move it to the body